v20.2/import-performance-best-practices.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Import Performance Best Practices
+summary: Best practices for optimizing import performance in CockroachDB.
+toc: true
+---
+
+This page provides best practices for optimizing [import](import.html) performance in CockroachDB.
+
+Import speed primarily depends on the amount of data the you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+
+- [Splitting data](#split-your-data-into-multiple-files)
+- [Import format](#choose-a-performant-import-format)
+
+{{site.data.alerts.callout_info}}
+If the import size is small (around 64-129 MiB), then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
```

Comment:
Do you know where this number comes from? It's pretty small and don't want to freak out users.